model_analyzer/result/result_manager.py

```diff
@@ -252,6 +244,38 @@ def _init_state(self):
         self._state_manager.set_state_variable('ResultManager.server_only_data',
                                                {})
 
+    def _complete_setup(self):
+        # The Report subcommand can init, but nothing needs to be done
+        if isinstance(self._config, ConfigCommandProfile):
+            self._complete_profile_setup()
+        elif isinstance(self._config, ConfigCommandAnalyze):
+            self._complete_analyze_setup()
```

Comment:
elif report:
  pass
else:
  raise exception

Reply:
Updated to add pass & exception